### PR TITLE
Paginate forward in results

### PR DESCRIPTION
**Pull request by bhasin85**
https://aodn.testrail.com/index.php?/cases/view/15

### CODE REVIEW

File: selenium/src/test/java/au/org/emii/portal/tests/step1/PaginateForwardInResults.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+public class PaginateForwardInResults extends BaseTest {
+
+    private static Logger log = Logger.getLogger(PaginateForwardInResults.class.getName());
+
+    @Test
+    public void paginationTest() throws InterruptedException {
+        log.info("Loading search page - Step 1");
+        getDriver().get(AODN_PORTAL_SEARCH_PAGE);
+
+        WebElement dataCollection = webElementUtil.findElement(By.xpath("//div[@class=\"resultsHeaderBackground\"][1]/div[1]/div[1]/h3"));
```
Comment (jonescc) [inline selenium/src/test/java/au/org/emii/portal/tests/step1/PaginateForwardInResults.java]: Why not match on resultsRowHeaderTitle?  it seems more direct

File: selenium/src/test/java/au/org/emii/portal/utils/PortalUtil.java [hunk 2]
```diff
@@ -32,4 +38,21 @@ public void validateFacetHeading(String facetHeading) {
...
     }
+
+    public void waitForSearchPanelReload(String oldDataCollectionText) {
+
+        int attempts = 1, totalAttempts = 2;
+        while(attempts <= totalAttempts) {
+            try {
+                WebDriverWait wait = new WebDriverWait(webElementUtil.driver, webElementUtil.TIMEOUT);
+                wait.until(ExpectedConditions.not(ExpectedConditions.textToBePresentInElement(
+                        webElementUtil.findElement(By.xpath("//div[@class=\"resultsHeaderBackground\"][1]/div[1]/div[1]/h3")), oldDataCollectionText)));
```
Comment (jonescc) [inline selenium/src/test/java/au/org/emii/portal/utils/PortalUtil.java]: And here

**Comment by jonescc** [comment]
Looks good - one minor comment only

**Comment by jonescc** [comment]
@bhasin85 - review finished - some minor comments as noted above.

**Comment by bhasin85** [comment]
@jonescc Fixed!

giphy

**Pull request merged**